ess/src/API/EMBC.ESS/Managers/Submissions/SubmissionsManager.cs

```diff
@@ -310,15 +310,22 @@ public async Task<VerifySecurityPhraseResponse> Handle(VerifySecurityPhraseQuery
 
         public async Task<string> Handle(SaveEvacuationFileNoteCommand cmd)
         {
+            if (string.IsNullOrEmpty(cmd.Note.Id) && string.IsNullOrEmpty(cmd.FileId)) throw new ArgumentNullException("NoteId and FileId cannot both be empty/null.");
+
             var note = mapper.Map<Resources.Cases.Note>(cmd.Note);
             var id = (await caseRepository.ManageCase(new SaveEvacuationFileNote { FileId = cmd.FileId, Note = note })).CaseId;
             return id;
         }
 
-        public async Task<EvacuationFileNotesQueryResult> Handle(EvacuationFileNotesQuery query)
+        public async Task<EvacuationFileNotesQueryResult> Handle(Shared.Contracts.Submissions.EvacuationFileNotesQuery query)
```

Comment:
can we maintain a convention of Response suffix for query return type, and Result suffix for a particular item in a response?

Reply:
in my defense that name was already there =)